packaging/docker/Dockerfile

```diff
@@ -24,7 +24,18 @@ ENV PATH=/usr/share/grafana/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bi
 WORKDIR $GF_PATHS_HOME
 
 RUN apk add --no-cache ca-certificates bash tzdata && \
-    apk add --no-cache --upgrade --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main openssl musl-utils
+    apk add --no-cache --upgrade --repository=http://dl-cdn.alpinelinux.org/alpine/edge/main openssl musl-utils && \
```

Comment:
You might want to only run this for the x86_64 build. I presume we'll only support running the plugin for that image?

Reply:
I think @xlson might be right.

Reply:
How do you usually do this, one `RUN` for x86_64 and one for ARM creating different layers or only additional layer for the x86_64?

Reply:
we are only supporting x86_64 for the plugin

Reply:
moved to its own block that tests for x86_64 similar to phantomjs